accessibility-checker-engine/src/v4/util/CSSUtil.ts

```diff
@@ -15,6 +15,7 @@
  *****************************************************************************/
 
 type PseudoClass = ":hover" | ":active" | ":focus" | ":focus-visible" | ":focus-within";
+import { DOMUtil } from "../../v2/dom/DOMUtil";
```

Comment:
Why was this added? I don't see it referenced anywhere in the file.

Reply:
it was a lftover, removed